tests/testrsa.nim

```diff
@@ -273,14 +273,14 @@ suite "RSA 512/1024/2048/4096 test suite":
     for i in 0..<TestsCount:
       var rkey1, rkey2: RsaPrivateKey
       var skey2 = newSeq[byte](4096)
-      var key = RsaPrivateKey.random(512)
-      var skey1 = key.getBytes()
+      var key = RsaPrivateKey.random(512).expect("random failed")
+      var skey1 = key.getBytes().expect("getBytes failed")
```

Comment:
the custom tends to be to describe why you expected it to work,ie, `expect("ok for key from random")`

Reply:
I honestly never liked that pattern :) the expected is obvious is the unexpected that is not.. should indeed be `unexpect`

Reply:
well, I guess it's mostly a matter of flow here how the code reads - I don't expect that random or getbytes failed, but rather that it worked ;) the `expect` function should then print the message accordingly: `operation failed, but shouldn't because the expectation was that: XXX`

more importantly though, the key is that I want to correlate the `getBytes` call with some logical progression of operations that leads me to believe that `getBytes` indeed shouldn't fail - some post-conditions of previous operations that work in tandem with preconditions on `getBytes` causing the composite operation to be safe.